### PR TITLE
feat: add missing bridge tokens

### PR DESCRIPTION
**Pull request by antonispoulakis**

### CODE REVIEW

**Review by ppoliani — APPROVED** [review]

File: cross-pool/tokens_metadata.json [hunk 1]
```diff
@@ -82,5 +82,17 @@
 		"name": "Kambria Yield Tuning Engine",
 		"symbol": "KYTE",
 		"decimals": 18
+	},
+	{
+		"address": "0x658e64ffcf40d240a43d52ca9342140316ae44fa",
+		"name": "OIN Finance",
```
Comment (philiparthurmoore) [inline cross-pool/tokens_metadata.json:88]: The official name for this token is `oinfinance` and not `OIN Finance`. I have asked them this many times and although it sounds quite silly they insist that the name is `oinfinance`. You can see that [here](https://bscscan.com/token/0x658e64ffcf40d240a43d52ca9342140316ae44fa#readContract) and [here](https://etherscan.io/token/0x9aeB50f542050172359A0e1a25a9933Bc8c01259).

Comment (philiparthurmoore) [reply cross-pool/tokens_metadata.json:88]: I apologize if there is confusion here - in Trust they have `OIN Finance` and in the actual token contract they have `oinfinance`. I have asked them twice about this and the answer they keep coming back with is `oinfinance`. However it appears that in Trust they are using the full name [here](https://github.com/trustwallet/assets/pull/7763/files). If it's irrelevant then I can ask them quickly again and be done with it - if it's super-important that this is accurate let me know. Just wanted to bring this issue up due to having the same problem during bridge deployment with them.

Comment (philiparthurmoore) [reply cross-pool/tokens_metadata.json:88]: Trust asset info on [master](https://github.com/DonXuShao/assets/blob/master/blockchains/ethereum/assets/0x9aeB50f542050172359A0e1a25a9933Bc8c01259/info.json).

Comment (antonispoulakis) [reply cross-pool/tokens_metadata.json:88]: that file is used for fetching the prices using lps and this name is only used internally. It won't appear anywhere like that

Comment (philiparthurmoore) [reply cross-pool/tokens_metadata.json:88]: Okay. Leave it as-is. Thank you.

**Pull request merged**